core/src/main/java/org/mule/endpoint/URIBuilder.java

```diff
@@ -255,25 +286,25 @@ private void appendMeta(StringBuilder buffer)
         }
     }
 
-    private OrderedQueryParameters appendAddress(StringBuilder buffer)
+    private OrderedQueryParameters appendAddress(StringBuilder buffer, Transformer tokenProcessor, Transformer tokenEncoder)
```

Comment:
Can the nopTransformer use all over be avoided somehow? Maybe by having another method that doesn't need another transformer?